### ISSUE TITLE
Scan ISBN/Camera doesn't work on Samsung Note10+ (Android 10)

### ISSUE THREAD

**Issue by QuAzI** [eleybourn/Book-Catalogue#844]
Restart doesn't help. Looks like no grants to access camera because I have never asked about it and can't find it in App rights settings.

![photo_2020-03-06_18-23-19](https://user-images.githubusercontent.com/1897605/76096719-99bb0780-5fd7-11ea-9b29-7a4cded9f298.jpg)

**Comment by shellyann76** [comment]
My S9 is having the same issue. Has something to do with the last update

**Comment by MtnViewJohn** [comment]
I got this message when I installed Pic2Shop and tried to use it from BookCatalogue. It was because Pic2Shop had not yet received the permissions that it needed to operate. When I ran Pic2Shop by itself, it asked for permissions; which I granted. After this, BookCatalogue was able to scan ISBN barcodes on my Pixel 3A with Android 10. Zxing does not work with this combination, even with permissions.
